lib/postprocessing/image.js

```diff
@@ -318,6 +324,12 @@ async function applyWatemark(img, intermediateRendition, rendition, directories)
 
 async function applyOperations(img, intermediateRendition, rendition, directories) {
     const instructions = rendition.instructions;
+
+    /*
+    ImageMagick supports color profiles, however, for images without a profile or a declaration of colorspace, ImageMagick assumes non-linear sRGB. Most image processing algorithms assume a linear colorspace, therefore it might be prudent to convert to linear color or remove the gamma function before certain image processing algorithms are applied.
+    */
+    // img.colorspace('RGB');
```

Comment:
@DrGirlfriend We'll need to uncomment that to use it - so we have the proper colors eventually.
But let's do this at the end, because the tests will need to be updated accordingly (since it influences the algorithms - especially resizing it seems).

Reply:
Done in https://github.com/adobe/asset-compute-sdk/pull/77.